### PR TITLE
Fix: Typos

### PR DESCRIPTION
**Pull request by datakabeta**
I came across some typos while using your documentation and figured I might as well quickly fix them instead of raising a support ticket.

I have fixed some typos across 4 files. That is all.

### CODE REVIEW

File: docs/quick-starts/experiments/analysis-plans
Comment (schmit) [inline docs/quick-starts/experiments/analysis-plans:1]: Not sure how this file got added, but I imagine it was not intended :)

Comment (datakabeta) [reply docs/quick-starts/experiments/analysis-plans:1]: Sorry! I think I know, but I'll save you the back story and delete it right away.

**Comment by emetelka** [comment]
Thanks for the contribution!

**Comment by datakabeta** [comment]
You're welcome!

**Pull request merged**